_episodes/02-installation.md

```diff
@@ -57,59 +57,47 @@ in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installa
 
 ## Install ESMValTool on Linux/MacOSX
 
-### Install Conda
+### Install Mamba
 
-ESMValTool is distributed using [Conda](https://conda.io/).
-Let's check if we already have Conda installed by running:
+ESMValTool is distributed using [Mamba](https://mamba.readthedocs.io/en/latest/index.html).
+To install mamba on ``Linux`` or ``MacOSX``, follow the  instructions below:
 
-```bash
-conda list
-```
-
-If conda is installed, we will see a list of packages.
-We recommend updating conda before esmvaltool installation. To do so, run:
-
-```bash
-conda update -n base conda
-```
-
-If conda is **not** installed, we can use Miniconda minimal installer for conda.
-We recommend a Python 3 based installer. For more information about installing conda,
-see [the conda installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
-
-To install conda on ``Linux`` or ``MacOSX``, follow the  instructions below:
-
-1.  Please download Miniconda3 at [the miniconda
-    page](https://docs.conda.io/en/latest/miniconda.html).
-    If you have problems with the 64 bit version in the next step(s)
-    you can alternatively try a 32 bit version.
+1.  Please download the installation file for the latest Mamba version [here](https://github.com/conda-forge/miniforge#mambaforge).
 
 2.  Next, run the installer from the place where you downloaded it:
 
     On ``Linux``:
 
     ```bash
-    bash Miniconda3-latest-Linux-x86_64.sh
+    bash Mambaforge-Linux-x86_64.sh
     ```
 
     On ``MacOSX``:
 
     ```bash
-    bash Miniconda3-latest-MacOSX-x86_64.sh
+    bash Mambaforge-MacOSX-x86_64.sh
     ```
 
 3.  Follow the instructions in the installer. The defaults should normally
     suffice.
 
 4.  You will need to restart your terminal for the changes to have effect.
 
-5.  Verify you have a working conda installation by listing all installed
-    packages
+5.  We recommend updating mamba before the esmvaltool installation. To do so, run:
+
+    ```bash
+    mamba update --name base mamba
+    ```
+
+6.  Verify you have a working mamba installation by: 
 
     ```bash
-    conda list
+    which mamba
```

Comment:
Should this return a value in the terminal, like the path? We need to add that I think.

Reply:
Thanks. I added that.